plugins/default/lib/mongodb_health.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import traceback
+from collections import Counter
+
+import openshift
+import nagios
+from mongodb import ReplsetStatuses
+
+
+def generate_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-s', '--selector', type=str, action="store",
```

Comment:
The reason for using the selector vs. using the service name is because we use service names in the core vs. mbaas, but the selector is the same in both cases.

@david-martin, @maleck13 - Is the mongodb service name different between an mbaas and core intentional?